lib/parser.rb

```diff
@@ -0,0 +1,15 @@
+class Parser
+  attr_accessor :file_pathname
+
+  def initialize(file_pathname)
+    @file_pathname = file_pathname
+  end
+
+  def get_first_line
+    if File.file?(@file_pathname)
```

Comment:
Como vc definiu o attr_accessor para o file_pathname, vc pode acessar sem o @.

```suggestion
    if File.file?(file_pathname)
```